test/types/bytes/basic.chpl

```diff
@@ -155,15 +155,15 @@ writeln(bytes_to_search.endsWith(b"start"));  //f
 writeln(bytes_to_search.endsWith(b"lorem", b"ipsum"));  //f
 writeln(bytes_to_search.endsWith(b"end"));   //t
 
-writeln(b.find(b"is")); //3
-writeln(b.rfind(b"is")); //6
+writeln(b.find(b"is")); //2
+writeln(b.rfind(b"is")); //5
 writeln(b.count(b"is")); //2
 
-writeln(b.find(b"is", region=4..)); //6
-writeln(b.rfind(b"is", region=..5)); //3
-writeln(b.count(b"is", region=..5)); //1
+writeln(b.find(b"is", region=3..)); //5
+writeln(b.rfind(b"is", region=..4)); //2
+writeln(b.count(b"is", region=..4)); //1
 
-writeln(b.find(b"is is")); //3
+writeln(b.find(b"is is")); //2
```

Comment:
github doesn't like files that it can't encode so I can't see the good file easily, but I hope it is updated w.r.t these

Reply:
The diff looks like this in lines 96-102:

old:
```
3                                                                               
6                                                                               
2                                                                               
6                                                                               
3                                                                               
1                                                                               
3                                                                               
```

vs.

new:
```
2                                                                               
5                                                                               
2                                                                               
5                                                                               
2                                                                               
1                                                                               
2                                                                               
```

Reply:
That's good